README.md

```diff
@@ -1,48 +1,133 @@
-# curriculum-databases-projects-template
+<!-- TABLE OF CONTENTS -->
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+# 📗 Table of Contents
 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-## Getting Started
+<!-- PROJECT DESCRIPTION -->
 
-This repository includes files with plain SQL that can be used to recreate a database:
+# 📖 [Vet Clinic] <a name="about-project"></a>
+
+> An database project on PostgreSQL.
+
+**[Vet Clinic]** A database project that involves using PostgreSQL. In this project, I used a relational database to create the data structure for a vet clinic
+
+## 🛠 Built With <a name="built-with"></a>
+
+- PostgreSQL
+- Github
+- SQL
+- GitFlow
+
+> Database
+
+<details>
+ <details>
+   <summary>Server</summary>
+   <ul>
+     <li>A localhost server created using PSQL</li>
+   </ul>
+ </details>
+
+ <!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- An Animal table containing the following columns: Id, Name, Date of birth, Escape attempts, Neutered, Weight
+
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Kindly fix the markdown format for key features.

Reply:
Done